### PR TITLE
6.2 release notes

### PR DESCRIPTION
**Pull request by jakelandis**

### CODE REVIEW

**Comment by jakelandis** [comment]
I based these changes from the most recent 6.1 release. Please review the diff https://github.com/elastic/logstash/pull/9085/commits/91fbc31214fe65fd451eab75ec2086ca7328f947 for the 6.2 changes.

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -11,7 +15,212 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-2-0]]
+=== Logstash 6.2.0 Release Notes
+
+* Added support to protect sensitive settings and configuration in a https://www.elastic.co/guide/en/logstash/current/keystore.html[keystore]
+* https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines] is now the default configuration (applicable for RPM/DEB/Docker installations)
```
Comment (andrewvc) [inline docs/static/releasenotes.asciidoc]: Can we embellish what this means for users? Will there be any breakage on upgrade? Do they need to take any new action WRT pipelines.yml?

Comment (andrewvc) [reply docs/static/releasenotes.asciidoc]: Put another way, is this a breaking change at all for these users.

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -14,6 +15,34 @@ See also:
...
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: +* Set default configuration for RPM/DEB/Docker installations to https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines]

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -14,6 +15,34 @@ See also:
...
+* Better defaults for higher throughput while under load. https://github.com/elastic/logstash/issues/8707[#8707] https://github.com/elastic/logstash/issues/8702[#8702]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: +* Set defaults to allow for higher throughput under load. https://github.com/elastic/logstash/issues/8707[#8707] https://github.com/elastic/logstash/issues/8702[#8702]

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -14,6 +15,34 @@ See also:
...
+* Log files now have a default max size and are compressed when rolled (applicable for ZIP based installs) https://github.com/elastic/logstash/issues/7482[#7482]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: +* Added default max size value (100MB) for log files.
+* Added compression when logs are rolled (for ZIP-based installs).

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: From 6.1 defaults to 6.2 defaults it is passive. ... If users deviated from the defaults and/or use some custom installation this _could_ , but not necessarily will break them. Most likely it should be passive. Here is the diff in question: https://github.com/elastic/logstash/commit/e12090667359044da4c746740dd9bebfb54dee87

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -14,6 +15,34 @@ See also:
...
+* Ability to specify --pipeline.id from the command line https://github.com/elastic/logstash/issues/8868[#8868]
+* Added new https://www.elastic.co/guide/en/logstash/current/plugins-filters-jdbc_static.html[JDBC static filter] as a default plugin
+* Continued improvements to the next generation of execution. Give it a try with the command line switch `--experimental-java-execution`
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: +* Implemented continued improvements to the next generation of execution. Give it a try with the command line switch `--experimental-java-execution.`

File: docs/static/releasenotes.asciidoc [hunk 6]
```diff
@@ -14,6 +15,34 @@ See also:
...
 endif::include-xpack[]
 
+[[logstash-6-2-0]]
+=== Logstash 6.2.0 Release Notes
+
+* Added support to protect sensitive settings and configuration in a https://www.elastic.co/guide/en/logstash/current/keystore.html[keystore]
+* https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines] is now the default configuration (applicable for RPM/DEB/Docker installations)
+* Better defaults for higher throughput while under load. https://github.com/elastic/logstash/issues/8707[#8707] https://github.com/elastic/logstash/issues/8702[#8702]
+* Log files now have a default max size and are compressed when rolled (applicable for ZIP based installs) https://github.com/elastic/logstash/issues/7482[#7482]
+* Ability to specify --pipeline.id from the command line https://github.com/elastic/logstash/issues/8868[#8868]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: +* Added ability to specify --pipeline.id from the command line https://github.com/elastic/logstash/issues/8868[#8868]

File: docs/static/releasenotes.asciidoc [hunk 7]
```diff
@@ -14,6 +15,34 @@ See also:
...
+* Added new https://www.elastic.co/guide/en/logstash/current/plugins-filters-jdbc_static.html[JDBC static filter] as a default plugin
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: +* Added https://www.elastic.co/guide/en/logstash/current/plugins-filters-jdbc_static.html[JDBC static filter] as a default plugin

File: docs/static/releasenotes.asciidoc (hunk 2 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: done with minor variant:  to *use* Multiple pipelines.

File: docs/static/releasenotes.asciidoc (hunk 3 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: done.

File: docs/static/releasenotes.asciidoc (hunk 4 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: done with minor variant : when log *files* are rolled

File: docs/static/releasenotes.asciidoc (hunk 5 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: done with minor variant - remove ending period to help prevent a copy/paste error

File: docs/static/releasenotes.asciidoc (hunk 6 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: done with minor variant: added ticks around `--pipeline.id`

File: docs/static/releasenotes.asciidoc (hunk 7 above)
Comment (jakelandis) [reply docs/static/releasenotes.asciidoc]: done.

**Comment by jakelandis** [comment]
@karenzone - thanks for the suggestions. All have been implemented, and any variation has been called out. 

You can view the changes applied here: https://github.com/elastic/logstash/pull/9085/commits/97bbb8b6ada8847a4bb195634d0d3723acea38e5

I also updated the order on this commit : https://github.com/elastic/logstash/pull/9085/commits/73ce8d2700884f12dad8b718c32739c013bc8dbe

**Comment by jakelandis** [comment]
@guyboertje - can you help to confirm that the content is correct ?

File: docs/static/releasenotes.asciidoc (hunk 3 above)
Comment (acchen97) [reply docs/static/releasenotes.asciidoc]: @jakelandis should we be explicit that this is for PQs?

**Pull request closed without merging**